xml/hwbp_registers.xml

```diff
@@ -378,6 +378,9 @@
             though the load will not update its destination register. Debuggers
             should consider this when setting such breakpoints on, for example,
             memory-mapped I/O addresses.
+
+            It is \unspecified what should happen when the instruction in question
+            performs multiple memory accesses.
```

Comment:
What should happen is only unspecified with respect to the memory accesses, not with respect to anything else.
```suggestion
            If an instruction matches this trigger and the instruction performs
            multiple memory accesses, it is \unspecified which memory accesses
            have completed before the trigger fires.
```